litmus-portal/Makefile

```diff
@@ -68,41 +68,60 @@ litmus-portal-push:
 
 
 .PHONY: litmus-portal-check
-litmus-portal-check : lint-checks unused-package-check
+litmus-portal-check : npm-install lint-checks unused-package-check
+
+npm-install:
+	@echo "------------------"
+	@echo "--> checking code style [frontend]"
+	@echo "------------------"
+	cd frontend && npm i
 
 lint-checks:
 	@echo "------------------"
-	@echo "--> checking code style"
+	@echo "--> checking code style [frontend]"
+	@echo "------------------"
+	cd frontend && npm run lint
+
+	@echo "------------------"
+	@echo "--> checking code style [backend]"
 	@echo "------------------"
 	@fmtRes=$$(gofmt -d $$(find . -path ./vendor -prune -o -name '*.go' -print)); \
 	if [ -n "$${fmtRes}" ]; then \
-		echo "gofmt checking failed!"; echo "$${fmtRes}"; echo; \
-		echo "Please ensure you are using $$($(GO) version) for formatting code."; \
+		echo "gofmt checking failed!" && echo "$${fmtRes}" \
+		&& echo "Please ensure you are using $$($(GO) version) for formatting code." \
+		&& exit 1;\
 	fi
 
 unused-package-check:
+	@echo "------------------"
+	@echo "--> Check litmus-portal frontend [depcheck]"
+	@echo "------------------"
+	cd frontend && npm i depcheck --no-save && node_modules/.bin/depcheck .
```

Comment:
Does this skip the missing dependency checks as it was not giving correct/desired output